From ea5d3db1ebbce51c9ebcd7f7162f0dd5f6399726 Mon Sep 17 00:00:00 2001
From: Steve Loughran <stevel@apache.org>
Date: Fri, 6 Feb 2015 20:59:56 +0000
Subject: [PATCH 1810/2848] HADOOP-11520. Clean incomplete multi-part uploads
 in S3A tests.  (Thomas Demoor via stevel)

(cherry picked from commit af7368cf032b445ff21c0b9dbb6605b28e4ca27c)

Change-Id: I5a2ab6e279e60c886b9f7a65563239ff97eded0b
---
 .../org/apache/hadoop/fs/s3a/S3ATestUtils.java     |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/S3ATestUtils.java b/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/S3ATestUtils.java
index 34603ea..2308dd4 100644
--- a/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/S3ATestUtils.java
+++ b/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/S3ATestUtils.java
@@ -46,6 +46,9 @@ public static S3AFileSystem createTestFileSystem(Configuration conf) throws
           "No test filesystem in " + TestS3AFileSystemContract.TEST_FS_S3A_NAME);
     }
     S3AFileSystem fs1 = new S3AFileSystem();
+    //enable purging in tests
+    conf.setBoolean(Constants.PURGE_EXISTING_MULTIPART, true);
+    conf.setInt(Constants.PURGE_EXISTING_MULTIPART_AGE, 0);
     fs1.initialize(testURI, conf);
     return fs1;
   }
-- 
1.7.9.5

